marklogic-data-hub/src/test/resources/entity-reference-model/entities/Customer.entity.json

```diff
@@ -9,6 +9,9 @@
       "required": [
         "name"
       ],
+      "rangeIndex": [
```

Comment:
Same thing here, shouldn't need this - should just be able to add sortable=true or facetable=true to customerId. That's important because as part of converting to HC, rangeIndex/pathRangeIndex/elementRangeIndex are all removed so that HC only deals with sortable and facetable.